test/html/external-in-template.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<!--
+    @license
+    Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>External Resources</title>
+  </head>
+  <body>
+    <dom-module id="my-element">
+      <template>
```

Comment:
Can you add a test that link to a .css file from two separate templates?

Reply:
I updated the test to demonstrate the same file is inlined in two places.